tests/operators/test_python.py

```diff
@@ -703,6 +703,25 @@ def test_clear_skipped_downstream_task(self):
             else:
                 raise ValueError(f'Invalid task id {ti.task_id} found!')
 
+    def test_xcom_push(self):
+        dag = DAG(
+            'shortcircuit_operator_test_xcom_push',
+            default_args={'owner': 'airflow', 'start_date': DEFAULT_DATE},
+            schedule_interval=INTERVAL,
+        )
+        short_op = ShortCircuitOperator(task_id='make_choice', dag=dag, python_callable=lambda: 'signature')
```

Comment:
I think this is not doing what you expect.
In `ShortCircuitOperator` the python_callable returns only True/False values.
If the value is False then the operator is looking for the downstream tasks and change their status to Skip.
It's not the same as `BranchPythonOperator` where the python callable is expected to return `task_ids`.